vb-fe/src/core/home/Home.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent } from "react";
+
+import { useSelector } from "react-redux";
+import { Flex, Heading } from "@chakra-ui/react";
+
+import { Navbar, Sheet, Profile, Search, Filters } from "components";
+import {
+  LayoutSidebar,
+  LayoutContent,
+  LayoutHeader,
+  LayoutBody,
+  LayoutWrapper,
+  LayoutFooter,
+  LayoutFooterOverlay,
+} from "layout/Layout";
+import { State } from "_store/rootReducer";
+
+import "core/home/Home.scss";
+
+interface Props {
+  y?: boolean;
+}
+
+export const Home: FunctionComponent<Props> = () => {
+  const userData = useSelector((state: State) => state.user.user);
+  const { bottomSheetExpanded } = useSelector((state: State) => state.system);
+
+  const profile = JSON.parse(localStorage.getItem("v-user") || "");
+  profile.displayName = profile.display_name;
+
+  const data = [];
+  for (let i = 0; i < 100; i += 1) {
+    data.push(i.toString());
+  }
+
+  const onSearch = (str: string): void => {
+    console.log("Search value: ", str);
```

Comment:
Should add a `TODO` here so we don't forget about this

Reply:
yeah but like functions cannot e empty - eslint :(